pkg/api/v1/types_swagger_doc.go

```diff
@@ -0,0 +1,1131 @@
+	}
+}
+
+func (PodStatusResult) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "PodStatusResult is a wrapper for PodStatus returned by kubelet that can be encode/decoded",
+		"metadata": "Standard object's metadata. See http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata",
+		"status":   "Most recently observed status of the pod. This data may not be up to date. Populated by the system. Read-only. See http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#spec-and-status",
+	}
+}
+
+func (PodTemplate) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "PodTemplate describes a template for creating copies of a predefined pod.",
+		"metadata": "Standard object's metadata. See http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata",
+		"template": "Template defines the pods that will be created from this pod template. http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#spec-and-status",
+	}
+}
+
+func (PodTemplateList) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "PodTemplateList is a list of PodTemplates.",
+		"metadata": "Standard list metadata.\n See http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#types-kinds",
+		"items":    "List of pod templates",
+	}
+}
+
+func (PodTemplateSpec) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "PodTemplateSpec describes the data a pod should have when created from a template",
+		"metadata": "Standard object's metadata. See http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata",
+		"spec":     "Specification of the desired behavior of the pod. See http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#spec-and-status",
+	}
+}
+
+func (Probe) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"": "Probe describes a liveness probe to be examined to the container.",
+		"initialDelaySeconds": "Number of seconds after the container has started before liveness probes are initiated. See http://releases.k8s.io/HEAD/docs/user-guide/pod-states.md#container-probes\"",
+		"timeoutSeconds":      "Number of seconds after which liveness probes timeout. Defaults to 1 second. See http://releases.k8s.io/HEAD/docs/user-guide/pod-states.md#container-probes",
+	}
+}
+
+func (RBDVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":          "RBDVolumeSource represents a Rados Block Device Mount that lasts the lifetime of a pod",
+		"monitors":  "CephMonitors is a collection of Ceph monitors. Required. See http://releases.k8s.io/HEAD/examples/rbd/README.md#how-to-use-it",
+		"image":     "RBDImage is the rados image name. Required. See http://releases.k8s.io/HEAD/examples/rbd/README.md#how-to-use-it",
+		"fsType":    "Filesystem type to mount. Must be a filesystem type supported by the host operating system. Ex. \"ext4\", \"xfs\", \"ntfs\" Required. See http://releases.k8s.io/HEAD/examples/rbd/README.md#how-to-use-it.",
```

Comment:
"Required: Filesystem type of the volume that you want to mount. Tip: Ensure that the filesystem type is supported by the host operating system. Examples: \"ext4\", \"xfs\", \"ntfs\". More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#rbd"